index.d.ts

```diff
@@ -1,128 +1,130 @@
-    | 'blok'
+    | "heading"
+    | "code_block"
+    | "paragraph"
+    | "blockquote"
+    | "ordered_list"
+    | "bullet_list"
+    | "list_item"
+    | "horizontal_rule"
+    | "hard_break"
+    | "image"
+    | "blok";
 
   type StoryblokRichtextMark =
-    | 'bold'
-    | 'italic'
-    | 'strike'
-    | 'underline'
-    | 'code'
-    | 'link'
-    | 'styled'
+    | "bold"
+    | "italic"
+    | "strike"
+    | "underline"
+    | "code"
+    | "link"
+    | "styled";
 
   type StoryblokRichtextContent = {
-    type: StoryblokRichtextContentType
+    type: StoryblokRichtextContentType;
     attrs?: {
-      level?: number
-      class?: string
-      src?: string
-      alt?: string
-      order?: number
+      level?: number;
+      class?: string;
+      src?: string;
+      alt?: string;
+      order?: number;
       body?: Array<{
-        _uid: string
-        Text: string
-        component: 'dia-tip'
-      }>
-    }
+        _uid: string;
+        Text: string;
+        component: "dia-tip";
+      }>;
+    };
     marks?: {
-      type: StoryblokRichtextMark
+      type: StoryblokRichtextMark;
       attrs?: {
-        href?: string
-        target?: string
-      }
-    }[]
-    text?: string
-    content: StoryblokRichtextContent[]
-  }
+        href?: string;
+        target?: string;
+      };
+    }[];
+    text?: string;
+    content: StoryblokRichtextContent[];
+  };
 
   type StoryblokRichtext = {
-    type: 'doc'
-    content: StoryblokRichtextContent[]
-  }
+    type: "doc";
+    content: StoryblokRichtextContent[];
+  };
 
-  export const NODE_HEADING = 'heading'
-  export const NODE_CODEBLOCK = 'code_block'
-  export const NODE_PARAGRAPH = 'paragraph'
-  export const NODE_QUOTE = 'blockquote'
-  export const NODE_OL = 'ordered_list'
-  export const NODE_UL = 'bullet_list'
-  export const NODE_LI = 'list_item'
-  export const NODE_HR = 'horizontal_rule'
-  export const NODE_BR = 'hard_break'
-  export const NODE_IMAGE = 'image'
+  export const NODE_HEADING = "heading";
+  export const NODE_CODEBLOCK = "code_block";
+  export const NODE_PARAGRAPH = "paragraph";
+  export const NODE_QUOTE = "blockquote";
+  export const NODE_OL = "ordered_list";
+  export const NODE_UL = "bullet_list";
+  export const NODE_LI = "list_item";
+  export const NODE_HR = "horizontal_rule";
+  export const NODE_BR = "hard_break";
+  export const NODE_IMAGE = "image";
 
-  export const MARK_BOLD = 'bold'
-  export const MARK_ITALIC = 'italic'
-  export const MARK_STRIKE = 'strike'
-  export const MARK_UNDERLINE = 'underline'
-  export const MARK_CODE = 'code'
-  export const MARK_LINK = 'link'
-  export const MARK_STYLED = 'styled'
+  export const MARK_BOLD = "bold";
+  export const MARK_ITALIC = "italic";
+  export const MARK_STRIKE = "strike";
+  export const MARK_UNDERLINE = "underline";
+  export const MARK_CODE = "code";
+  export const MARK_LINK = "link";
+  export const MARK_STYLED = "styled";
+
+  export type RenderOptions = {
```

Comment:
I'd vouch for using `interface` over `type` here, since `RenderOptions` essentially is an object type declaration which might be extended by a user.

> An interface declaration is another way to name an object type. [...] Type aliases and interfaces are very similar, [...] the key distinction is that a type cannot be re-opened to add new properties vs an interface which is always extendable. – [Typescriptlang.org](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#interfaces)

```suggestion
  export interface RenderOptions {
```